src/main/java/com/jfrog/ide/idea/ui/webview/event/model/IdeEvent.java

```diff
@@ -0,0 +1,22 @@
+package com.jfrog.ide.idea.ui.webview.event.model;
+
+/**
+ * Represents an IDE-specific event that can be sent from the IDE to the Webview.
```

Comment:
Isn't it sent from the webview to the IDE?
It seems to be the other way around also in WebviewEvent.
Or did I just get it wrong?

Reply:
You are correct, i will switch between both ->
```
 Represents a Webview-specific event that can be sent from the IDE to the Webview.
```